config/src/test/java/org/springframework/security/config/annotation/web/configurers/NamespaceHttpX509Tests.java

```diff
@@ -21,13 +21,11 @@
 import java.security.cert.CertificateFactory;
 import java.security.cert.X509Certificate;
 
-import javax.security.auth.x500.X500Principal;
 import javax.servlet.http.HttpServletRequest;
 
-import org.bouncycastle.asn1.x500.X500Name;
-import org.bouncycastle.asn1.x500.style.BCStyle;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import sun.security.x509.X500Name;
```

Comment:
Can you clarify how these changes pertain to the PR?

Reply:
Those changes came from the revert that I did to the old commit, where we weren't using toolchain to build the modules